### PR TITLE
Consistent score on all difficulties

### PR DESCRIPTION
**Pull request by ComputasAlex**
Made a small formula which will secure consistent score independent of length of round and max score.
Also refactored to have single variables that get used throughout.

Up for discussion: Increase maxScore?

### CODE REVIEW

File: src/app/game/game-draw/game-draw.component.ts [hunk 1]
```diff
@@ -273,7 +273,7 @@ export class GameDrawComponent implements OnInit, OnDestroy {
   private createDrawingTimer() {
     return new Observable((observer) => {
       let color = 'red';
-      const intervalDuration = 100;
+      const intervalDuration = this.gameConfigService.getConfig.intervalDuration;
```
Comment (vkallerud) [inline src/app/game/game-draw/game-draw.component.ts]: Maybe make a variable 'configValues' outside the method, and refer to that in here. This is good for scalability

File: src/app/game/services/game-config.service.ts [hunk 2]
```diff
@@ -68,9 +72,11 @@ export class GameConfigService {
   }
 
   getScoreSettings(): ImageScoreConfig {
+    const maxScore = 333;
+    const intervalDuration = this.getConfig.intervalDuration;
     return {
-      maxScore: 333,
-      scoreDecrement: 1.67336683417,
+      maxScore: maxScore,
+      scoreDecrement: maxScore / ((this.getConfig.secondsPerRound * 1000) / intervalDuration),
```
Comment (vkallerud) [inline src/app/game/services/game-config.service.ts:79]: Nice!!

**Review by vkallerud — CHANGES_REQUESTED** [review]

File: src/app/game/game-draw/game-draw.component.ts
Comment (vkallerud) [inline src/app/game/game-draw/game-draw.component.ts:1]: 🫱🏻‍🫲🏼

**Review by vkallerud — APPROVED** [review]
🫱🏻‍🫲🏼

File: src/app/game/services/game-config.service.ts (hunk 2 above)
Comment (ComputasAlex) [reply src/app/game/services/game-config.service.ts:79]: thank you

**Pull request merged**